runner/ansible/roles/2.2.8/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.2.8'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    #  HANA and SPS versions are compatible
+    sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:upper:]' '[:lower:]')
```

Comment:
Careful with this `sudo`, we shouldn't have it here if possible

Reply:
Agreed, but I think that command does need root privs. We can look at https://docs.ansible.com/ansible/2.4/become.html for this kind of thing, but I would need to be on my primary system to look into to that.